src/test/scala/io/iohk/ethereum/db/dataSource/LevelDBDataSourceTest.scala

```diff
@@ -1,15 +1,22 @@
 package io.iohk.ethereum.db.dataSource
 
+import java.nio.file.Files
+
 import org.scalatest.FlatSpec
 
 class LevelDBDataSourceTest extends FlatSpec with DataSourceTestBehavior {
```

Comment:
this test is now failing:
`Error during tests:
[error] 	io.iohk.ethereum.db.dataSource.LevelDBDataSourceTest`